models/user.models.js

```diff
@@ -1,7 +1,9 @@
 const db = require("../db/connection")
 
 
-exports.fetchUsers = async ()=>{
-    const users = await db.query("SELECT * FROM users;")
-    return users.rows
+exports.fetchUsers = ()=>{
+    return db.query("SELECT * FROM users;").then(({rows})=>{
+        return rows
+    })
+    
```

Comment:
Any particular reason you went back to promises here Hammam?
It's entirely your preference, just interested!